source/common/formatter/substitution_formatter.h

```diff
@@ -228,6 +228,33 @@ class HeaderFormatter {
   absl::optional<size_t> max_length_;
 };
 
+/**
+ * FormatterProvider for headers byte size.
+ */
+class HeadersByteSizeFormatter : public FormatterProvider {
+public:
+  // TODO(htuch): Add RequestTrailers here.
```

Comment:
This is not a TODO for me :) Please either file an issue and reference it by number or assign to yourself.

Reply:
Done.